charts/ggscout/templates/clusterrole.yaml

```diff
@@ -13,4 +13,14 @@ rules:
   - apiGroups: [""]
     resources: ["secrets"]
     verbs: ["get", "list"]
+  - apiGroups: [ "" ]
+    resources: [ "configmaps" ]
+    verbs: [ "get", "list" ]
+  - apiGroups: [ "" ]
```

Comment:
`deployments` is in `app` `apiGroups` I think

Reply:
`deployments                         deploy                              apps/v1                                   true         Deployment`

Reply:
Nice catch